src/components/Home/reducers/launchReducer.ts

```diff
@@ -1,35 +1,54 @@
-import { AnyAction } from '@reduxjs/toolkit';
 import { Launches } from '../constants/launches';
 import { LaunchActionTypes } from '../types/launchActionTypes';
+import {
+  LaunchAction,
+  LaunchActionError,
+  LaunchActionRequest,
+  LaunchActionSuccess,
+} from '../types/LaunchActions';
+import type { Launch } from '../../../shared/api/types/Launch';
 import type { LaunchState } from '../types/LaunchState';
 
-const initialState: LaunchState = {
-  [Launches.UPCOMING]: [],
-  [Launches.PREVIOUS]: [],
-  [Launches.CREWED]: [],
-  [Launches.CALIFORNIA]: [],
-  [Launches.FLORIDA]: [],
+const initState: LaunchState = {
+  launches: [],
+  launchAmount: 0,
   isLoading: false,
+  isScrollLoading: false,
   error: '',
```

Comment:
A common alert will be used, so the error does not need here

Reply:
deleted